releasenotes/notes/add-upper-constraints-to-es-c2eb0b51c83e30e2.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The elasticsearch role now respects upper constraints when installing
+    pip packages. The upper constriants file is defined in
+    ``rpcd/playbooks/group_vars/all.yml``.
```

Comment:
Is it? AFAIK it uses the OSA upper constraints setting, which uses the upstream requirements repo until the repo server exists, when it switches to using the repo's compiled/artifacted constraints.